.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# PULL REQUEST TEMPLATE
+
+## Description
+> Please provide a summary of your changes here. 
+> * Which issue is this pull request belonging to? (*Refer to issue here*)
```

Comment:
We should probably point out to follow the syntax for [auto-closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords) here.

As a site note, I personally prefer to have issue references in the commit messages, not in the PR description / discussion.